cmd/analyzer/analyzer.go

```diff
@@ -94,6 +94,7 @@ func parseFlags(version string) cliFlags {
 	fs.StringVar(&cli.viewsOpts.ViewershipEventsTable, "viewership-events-table", "livepeer-analytics.viewership.staging_viewership_events", "BigQuery table to read viewership events from")
 	fs.StringVar(&cli.viewsOpts.ViewershipSummaryTable, "viewership-summary-table", "livepeer-analytics.viewership.staging_viewership_summary_by_video", "BigQuery table to read viewership summarized metrics from")
 	fs.StringVar(&cli.usageOpts.HourlyUsageTable, "hourly-usage-table", "livepeer-analytics.staging.hourly_billing_usage", "BigQuery table to read hourly usage metrics from")
+	fs.StringVar(&cli.usageOpts.ExplorerDayDataTable, "explorer-data-table", "livepeer-analytics.staging.explorer_day_data", "BigQuery table to read total usage metrics from")
```

Comment:
for consistency with the existing flags wdyt
```suggestion
	fs.StringVar(&cli.usageOpts.DailyUsageTable, "daily-usage-table", "livepeer-analytics.staging.explorer_day_data", "BigQuery table to read total usage metrics from")
```

Conceptually this code doesn't know who's gonna use the API (explorer), it only knows what the API/data is (total/daily), right?